examples/elasticsearch/es-rc.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: es
+  namespace: default
+  labels:
+    component: elasticsearch
+spec:
+  replicas: 1
+  selector:
+    component: elasticsearch
+  template:
+    metadata:
+      labels:
+        component: elasticsearch
+    spec:
+      serviceAccount: elasticsearch
+      containers:
+      - name: es
+        image: quay.io/pires/docker-elasticsearch-kubernetes:1.7.1-4
```

Comment:
@brendandburns @a-robinson Do you prefer images for examples to come from `gcr.io` or is this location OK? I don't have any problems with it personally, but not sure quite you expect from contributed examples.